### PR TITLE
DOCS-9373 updating deploy topic.

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

File: modules/ROOT/pages/deploy-to-runtime-fabric.adoc [hunk 1]
```diff
@@ -10,11 +10,12 @@ Deploy a Mule application to Anypoint Runtime Fabric and test the inbound traffi
 Before deploying an application to Anypoint Runtime Fabric, you must perform the following:
 
 * Install and configure Anypoint Runtime Fabric. See xref:install-create-rtf-arm.adoc[Begin Runtime Fabric Installation] for more information.
-* Ensure that Runtime Fabric is associated with an environment.
+* Ensure that Runtime Fabric is associated with an environment. See xref:associate-environments.adoc[Associate Business Groups and Environments to Anypoint Runtime Fabric]
 * Determine the resource allocation requirements for your application. See xref:deploy-resource-allocation.adoc[Determine Application Resource Allocation on Runtime Fabric] for more information.
```
Comment (sleslie23) [inline modules/ROOT/pages/deploy-to-runtime-fabric.adoc:15]: @sundippatel @sarjeet2013 Is this only applicable to appliance?

File: modules/ROOT/pages/deploy-to-runtime-fabric.adoc [hunk 2]
```diff
@@ -10,11 +10,12 @@ Deploy a Mule application to Anypoint Runtime Fabric and test the inbound traffi
...
 * Your Anypoint user may require the *Exchange Contributors* permission to deploy applications to Runtime Fabric.
+* To deploy an application to a specific domain (endpoint), ensure that you have setup domains (endpoints) in your environment. See xref:enable-inbound-traffic-self.adoc[Enable Inbound Traffic on Anypoint Runtime Fabric (Self-managed Kuberenetes)] or xref:enable-inbound-traffic.adoc[Enable Inbound Traffic on Anypoint Runtime Fabric (VM / Baremetal)]
```
Comment (sleslie23) [inline modules/ROOT/pages/deploy-to-runtime-fabric.adoc:17]: @sundippatel @sarjeet2013 We use "domain" when deploying an app, but "endpoint" when enabling inbound traffic. Is there a conceptual/technical difference? Or are they referring to the same thing?

Comment (sleslie23) [reply modules/ROOT/pages/deploy-to-runtime-fabric.adoc:17]: Also, Shayan mentioned that we prefer that users use domains. Let me know if we need to call this out more.

**Review by sundippatel — COMMENTED** [review]
Lines 37-43: This seems to conflict with the procedure later on (Lines 66-69). I think can remove 37-43 as a result.

Line 62: Trim " by adjusting the sliders to the appropriate values" -- it's outdated with recent versions of the UX.

Line 74 will depend on whether on self-managed K8s or VMs/bare metal: 
*For self-managed:* To send requests to a deployment using a different supported endpoint, add additional endpoints under the Inbound Traffic setting on this Runtime Fabric. 
*For VMs / bare metal:* Additional endpoints are sourced from the subject alternative names (SAN) certificate property. If a certificate has multiple DNS entries specified in the SAN property, the available URLs are displayed in the *Application url* section.

File: modules/ROOT/pages/deploy-to-runtime-fabric.adoc [hunk 3]
```diff
@@ -50,25 +55,34 @@ This tab displays the name and type of the deployment target you selected above.
...
 +
 [NOTE]
 To access a Runtime Fabric instance using more than one DNS, add additional DNS entries in the subject alternative names (SAN) certificate property. If a certificate has multiple DNS entries specified in the SAN property, the available URLs are displayed in the *Application url* section.
-* *Enable Last-Mile security* to forward HTTPS connections to be decrypted by this application. This requires an SSL certificate to be included in the Mule application, and also requires more CPU resources.
++
+* *Enable Last-Mile security* to forward HTTPS connections to be decrypted by this application. This requires an SSL certificate to be included in the Mule application, and also requires more CPU resources. This option is only applicable to Runtime Fabric on VMs / baremetal.
++
+* *Forward SSL session* This option is only applicable to Runtime Fabric on VMs / baremetal.
```
Comment (sleslie23) [inline modules/ROOT/pages/deploy-to-runtime-fabric.adoc:85]: This option is only available on VM / bare metal, but is missing from the current docs. Can someone provide info?

File: modules/ROOT/pages/deploy-to-runtime-fabric.adoc (hunk 2 above)
Comment (sleslie23) [reply modules/ROOT/pages/deploy-to-runtime-fabric.adoc:17]: After my with Rohit, I think this is resolved.

File: modules/ROOT/pages/deploy-to-runtime-fabric.adoc (hunk 3 above)
Comment (sleslie23) [reply modules/ROOT/pages/deploy-to-runtime-fabric.adoc:85]: Info provided in slack channel.

File: modules/ROOT/pages/deploy-to-runtime-fabric.adoc (hunk 1 above)
Comment (sleslie23) [reply modules/ROOT/pages/deploy-to-runtime-fabric.adoc:15]: Resolved.

**Review by sundippatel — APPROVED** [review]

**Pull request merged**